api/src/test/resources/application.properties

```diff
@@ -9,9 +9,9 @@ spring.liquibase.enabled=false
 spring.main.allow-bean-definition-overriding=true
 
 # Avoid log spam in tests
-spring.main.log-startup-info=OFF
+spring.main.log-startup-info=false
 spring.main.banner-mode=off
-logging.level.org.springframework=WARN
+logging.level.org.springframework=DEBUG
```

Comment:
please revert this to WARN, we don't want to flood the logs

Reply:
ah, okay done. (but there is unit test, not production server)